documentation/objects/objects.md

```diff
@@ -209,10 +202,25 @@ Assuming our labels were correct this will lead to a good object classification:
 
 ## Export
 
-In the [Export Applet][] you can export the following images: "Object Predictions", "Object Probabilities" "Blockwise Object Predictions".
-In addition to the image export, it is also possible to generate a table that encompasses all information about the objects used during classification.
-Table configuration can be accessed with the _Configure Feature Table Export_ button.
-In this new window there are three vertical tabs:
+### Regular image exports
+
+In the [Export Applet][] you can export:
+* Object Predictions: An image where all pixels belonging to an object have the value of the object's most probable category (1 for the first label category, 2 for the second, etc.)
+* Object Probabilities: An image where all pixels belonging to an object have the probability of the object belonging into its most probable category.
+* Blockwise Object Predictions/Probabilities: Same as above, but perform the computation on the input images in blocks of the size specified in the "Blockwise" applet (see below). This makes it possible to process images that are larger than the machine's RAM.
+* Object Identities: An image where all pixels belonging to an object have the value of the object's id (1 for the first object, 2 for the second, etc.)
+
+### Object feature table export
+
+In addition to the image export, it is also possible to generate a table that encompasses all information about the objects used during classification. To activate this export:
+1. Access the table configuration with the _Configure Feature Table Export_ button. See below for more details on the configuration options.
+2. Change the file name and choose your preferred format.
+3. Choose the features you want to include in the table.
+4. Confirm with OK.
+
+Any regular image export (predictions, probabilities) will now also generate the table file according to the table export settings.
+
+The table export can be configured in three sections:
```

Comment:
maybe also add that the table will not be available in block-wise mode.

Reply:
It still exports a table just fine in blockwise mode - though it seems to only contain a list of the objects, probabilities and bounding boxes, missing the feature values:

![image](https://github.com/ilastik/ilastik.github.io/assets/63287233/92812d96-1740-42fd-8f24-694fef534f5f)

Reply:
hm we should confirm this for the headless mode. I don't think it would be possible to have the table in that setting, let alone continuous object ids with our current implementation.

Reply:
I vaguely remember our discussion relating to this issue (the table basically breaking blockwise export, as the table  will require processing of the whole image). Did we come to a conclusion on how to proceed?

1.) Document this behavior (blockwise + table will not work for data that does not fit into ram)
2.) Fix it on the ilastik side (no table output for blockwise)
3.) dream solution: make object classification lazy :)

Reply:
I don't actually know what the behaviour in any of the questionable scenarios is. It would be good to investigate what happens to the table export with images larger than RAM in blockwise export or headless mode and document it, but I'm not really up for that effort right now. Hence why there is still no mention of it.